bmc/redfish_local.go

```diff
@@ -5,16 +5,25 @@ package bmc
 
 import (
 	"context"
+	"errors"
 	"fmt"
+	"strings"
 
 	"github.com/stmcginnis/gofish/redfish"
 )
 
 var _ BMC = (*RedfishLocalBMC)(nil)
 
+var defaultMockedBIOSSetting = []map[string]any{
+	{"name": "abc", "type": "string", "reboot": false, "value": "blah"},
+	{"name": "fooreboot", "type": "integer", "reboot": true, "value": 123},
+}
```

Comment:
It seems that `defaultMockedBIOSSetting` is mostly indexed by the `name` property by iterating, which has linear time complexity. Would it make to use a `map[string]map[string]any`, where the `name` of a setting is the key, e.g. `{"abc": {"type": "string", "reboot": false, "value": "blah"}}`? If this is very inconvenient at API boundaries, we can keep for the mock.